README.md

```diff
@@ -1,2 +1,4 @@
 # code-review-test-repo
 A repo to test code review actions.
+
```

Comment:
Add a comment as part of an actual review.

Reply:
This test review was by author. This is a comment added by "Add single comment". Next, a collaborator will make a review.